x-pack/plugins/security_solution/common/endpoint/models/event.ts

```diff
@@ -32,9 +32,9 @@ export function eventName(event: ResolverEvent): string {
   }
 }
 
-export function eventId(event: ResolverEvent): string {
+export function eventId(event: ResolverEvent): number | undefined | string {
```

Comment:
@jonathan-buttner this function was converting the serial event id to a string. if it was undefined or 0 it was being converted to `''`. i don't know the reasoning, but i undid it. some BE code was effected. let me know if this is OK

Reply:
@bkimmel or @kqualters-elastic may have thoughts here.

Reply:
I think this is fine I left some comments about checking for `undefined` and not displaying the string `'undefined'`.